esapi/errors.py

```diff
@@ -205,3 +204,11 @@ class ESQueryTooManyDestinationZonesError(FondefVizError):
     def __init__(self, limit_number):
         message = 'No puede seleccionar más de {0} zonas de destino'.format(limit_number)
         super(ESQueryTooManyDestinationZonesError, self).__init__(419, message)
+
+
+class ESQueryAuthRouteCodeTranslateDoesNotExists(FondefVizError):
+    """ It raises when user send an auth route code that doesnt exists """
```

Comment:
falta el apostrofe `doesn't` o poner `does not`. Entiendo que el verbo es `exist`, la s es para la tercera persona en presente, corregir (creo que en otros lados también está mal escrita)

Reply:
Ok, revisaré para corregir los otros.